showcase/tests/integration/components/hds/time/index-test.js

```diff
@@ -0,0 +1,304 @@
+      navigator.lagnguage,
+      options.displayFormat
+    );
+    assert.dom('.hds-time').hasText(expectedDateString);
+    // test tooltip content
+    assert.dom('.tippy-content').hasText(this.defaultDate.toISOString());
+  });
+
+  // default using ISO date string
+  test('it should render the correct string in the default format when passing an ISO date string', async function (assert) {
+    await render(hbs`
+      <Hds::Time @date="05 September 2018 14:07:32" @isOpen={{true}} />
+    `);
+    let options = DEFAULT_DISPLAY_MAPPING[DISPLAY_KEY_FRIENDLY_LOCAL];
+    let expectedDateString = new Date(
+      '05 September 2018 14:07:32'
+    ).toLocaleString(navigator.lagnguage, options.displayFormat);
+    assert.dom('.hds-time').hasText(expectedDateString);
+    // test tooltip content
+    assert
+      .dom('.tippy-content')
+      .hasText(new Date('05 September 2018 14:07:32').toISOString());
+  });
+
+  // Friendly display types
+
+  // friendly local using Date object
+  test('it should render the correct string in friendly local format when passing a Date object', async function (assert) {
+    this.set('friendlyLocalDate', new Date('05 September 2018 14:07:32'));
+    await render(hbs`
+      <Hds::Time @date={{this.friendlyLocalDate}} @display="friendly-local" @isOpen={{true}} />
+    `);
+    let options = DEFAULT_DISPLAY_MAPPING[DISPLAY_KEY_FRIENDLY_LOCAL];
+    let expectedDateString = new Date(this.friendlyLocalDate).toLocaleString(
+      navigator.lagnguage,
+      options.displayFormat
+    );
+    assert.dom('.hds-time').hasText(expectedDateString);
+    // test tooltip content
+    assert.dom('.tippy-content').hasText(this.friendlyLocalDate.toISOString());
+  });
+
+  // friendly local using ISO date string
+  test('it should render the correct string in friendly local format when passing an ISO date string', async function (assert) {
+    await render(hbs`
+      <Hds::Time @date="05 September 2018 14:07:32" @display="friendly-local" @isOpen={{true}} />
+    `);
+    let options = DEFAULT_DISPLAY_MAPPING[DISPLAY_KEY_FRIENDLY_LOCAL];
+    let expectedDateString = new Date(
+      '05 September 2018 14:07:32'
+    ).toLocaleString(navigator.lagnguage, options.displayFormat);
+    assert.dom('.hds-time').hasText(expectedDateString);
+    // test tooltip content
+    assert
+      .dom('.tippy-content')
+      .hasText(new Date('05 September 2018 14:07:32').toISOString());
+  });
+
+  // friendly only using Date object
+  test('it should render the correct string in friendly only format when passing a Date object', async function (assert) {
+    this.set('friendlyOnlyDate', new Date('05 September 2018 14:07:32'));
+
+    await render(hbs`
+      <Hds::Time @date={{this.friendlyOnlyDate}} @display="friendly-only" @isOpen={{true}} />
+    `);
+    assert.dom('.hds-time').hasText('Sep 5, 2018');
+    // test tooltip content
+    assert.dom('.tippy-content').hasText(this.friendlyOnlyDate.toISOString());
+  });
```

Comment:
unfortunately, the friendly date format is locale-dependent, meaning the output is not always `Sep 5, 2018`. for example, when I run this test (and a few others) on my machine it renders `5 Sept 2018`

Reply:
pushed 91b9f6e to resolve this issue by overriding the locale to 'en-US'